### DISCUSSION TITLE
Disable sleep function

### DISCUSSION THREAD

**Discussion by PeterPedro** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#248] [Ideas]
Just an idea, it would be nice to have it keep the computer ON, as long as it's running, just like an open video window keeps a computer running. That way it's not necessary to change the computer's settings every time you run the script.

**Comment by PeterPedro** [comment]
Found this, it's supposed to be cross-platform but I haven't tried it yet

```
from wakepy import set_keepawake, unset_keepawake

set_keepawake(keep_screen_awake=False)
# do stuff that takes long time
unset_keepawake()
```
Source: (https://stackoverflow.com/questions/57647034/prevent-sleep-mode-python-wakelock-on-python)

**Comment by Rakambda** [comment]
In my opinion it shouldn't really be the software that should do that. I'd prefer setting up manually the device its running on as a user device or server depending on the needs, but not having a software handle that.

What if you have a flag set and the process crash ? Would leave the flag on and your comp doesn't go to sleep as it should ?

Just giving my opinion though.
